styles.css

```diff
@@ -91,6 +95,331 @@ ul li a {
   font-family: Arial, sans-serif, 'Roboto';
 }
 
-.illustration {
-  display: none;
+.works {
+  display: grid;
+  grid-template-rows: auto auto auto auto auto auto auto auto;
+  row-gap: 10px;
+  justify-items: center;
+  margin: 0;
+}
+
+.works h2 {
+  color: #1c1a19;
+  font-family: Arial, sans-serif, 'Crete Round';
+  font-size: 40px;
+  font-weight: 400;
+  font-style: normal;
+  padding: 100px 24px 10px;
+}
+
+.works img {
+  padding: 25px;
+}
+
+.stories-intro {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.stories-intro img {
+  width: 100%;
+}
+
+.stories-intro h2 {
+  font-size: 32px;
+  font-weight: 400;
+  color: #091e42;
+  font-family: 'Crete Round', arial, sans-serif;
+  align-self: flex-start;
+  padding: 0 20px;
+}
+
+.stories-intro p {
+  font-weight: 400;
+  font-size: 16px;
+  color: #091e42;
+  font-family: arial, sans-serif, 'Inter';
+  line-height: 24px;
+  padding: 0 20px;
+}
```

Comment:
- [ ] Please 🙏🏽 align the content of the card to match the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219761724-06cc7ddc-d44a-417a-860a-2430a32b32b0.png) | ![image](https://user-images.githubusercontent.com/58642949/219762040-93d026b4-fd2e-43a5-8ae4-1e8dbeeb843a.png) |